### ISSUE TITLE
No super admin

### ISSUE THREAD

**Issue by bas-ie** [chriszarate/docker-wordpress-vip#4]
Hi Chris, and thanks for your work on this.

I'm not a frequent WordPress developer, so I may be missing something obvious but the wordpress/wordpress user does not appear to have super admin privileges. Is there a way to add a SA or promote the default user?

Edit: I should mention, I tried:
```
docker-compose exec wordpress wp super-admin add wordpress
```
which results in:
```
PHP Warning:  in_array() expects parameter 2 to be array, string given in phar:///usr/local/bin/wp-cli/php/commands/super-admin.php on line 110
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/wp-cli:0
PHP   2. include() /usr/local/bin/wp-cli:4
PHP   3. include() phar:///usr/local/bin/wp-cli/php/boot-phar.php:5
PHP   4. WP_CLI\Runner->start() phar:///usr/local/bin/wp-cli/php/wp-cli.php:21
PHP   5. WP_CLI\Runner->_run_command() phar:///usr/local/bin/wp-cli/php/WP_CLI/Runner.php:906
PHP   6. WP_CLI\Runner->run_command() phar:///usr/local/bin/wp-cli/php/WP_CLI/Runner.php:319
PHP   7. WP_CLI\Dispatcher\Subcommand->invoke() phar:///usr/local/bin/wp-cli/php/WP_CLI/Runner.php:312
PHP   8. call_user_func:{phar:///usr/local/bin/wp-cli/php/WP_CLI/Dispatcher/Subcommand.php:372}() phar:///usr/local/bin/wp-cli/php/WP_CLI/Dispatcher/Subcommand.php:372
PHP   9. WP_CLI\Dispatcher\CommandFactory::WP_CLI\Dispatcher\{closure}() phar:///usr/local/bin/wp-cli/php/WP_CLI/Dispatcher/Subcommand.php:372
PHP  10. call_user_func:{phar:///usr/local/bin/wp-cli/php/WP_CLI/Dispatcher/CommandFactory.php:67}() phar:///usr/local/bin/wp-cli/php/WP_CLI/Dispatcher/CommandFactory.php:67
PHP  11. Super_Admin_Command->add() phar:///usr/local/bin/wp-cli/php/WP_CLI/Dispatcher/CommandFactory.php:67
PHP  12. in_array() phar:///usr/local/bin/wp-cli/php/commands/super-admin.php:110
No changes.

```

**Comment by bas-ie** [comment]
Used https://drawne.com/add-super-admin-wordpress-network/ to work around:

```
docker ps
# find the mariadb process
docker exec -i -t abcde123 /bin/bash
```

Then:

```
mysql -u wordpress -p

MariaDB> use wordpress;
MariaDB> select * from wp_sitemeta where meta_key="site_admins";
```
`meta_value` should be set to something like `a:1:{i:0;s:9:"wordpress";}`. Just in case anyone else has a similar issue.

**Comment by chriszarate** [comment]
Thanks for the report. I fixed this in 72d12fba1ff7c45517ab4530afb4809b9e76fde0. The issue is that I wasn't properly customizing WP-CLI before its first use in `docker-entrypoint.sh`. Therefore if you changed the URL from `project.dev`, you ran into this issue.

Pull down the latest changes from this repo and then run:

```
docker-compose stop # stop any running stacks
docker-compose build
docker-compose up -d # restart stacks
```
